### ISSUE TITLE
Splitting topics into sub categories

### ISSUE THREAD

**Issue by vedanthv** [DaedalusDreamJournal/66daysofdata#9]
@DaedalusDreamJournal pointed out that there was too much math theory inn one category which can be overwhelming. I have changed the structure of the curriculum in this Google Doc with the proposed changes
https://docs.google.com/document/d/1WN35mSfNXLQc6Dbg-V3GCWIpfgipyyEWHcHVcXngvB0/edit
If you have any changes to make, please go ahead but leave a comment here explaining the changes.

**Comment by DaedalusDreamJournal** [comment]
yes, it does make more sense like this. I will make a pull request later with the proposed changes from the google doc to the roadmap. I do really like some of the current organization in the roadmap in google currently, especially having mat/stats followed by the matplotlibs and the math/linear algebra followed by Machine learning fundamental.

This makes me think I might add a proposal to add text in between major sections (for example math-> matplolib) explaining for example how the current module builds on the previous one or/and vice versa.

**Comment by vedanthv** [comment]
Sure. We can do that. Are you satisfied with the structure of topics. Shall I work on the reorganization part from tomorrow?
